pkg/cli/admin/upgrade/status/examples/4.16.0-ec2-control-plane-updated-pdb-prohibits-draining.detailed-output

```diff
@@ -152,6 +146,15 @@ Message: Node build0-gstfj-ci-tests-worker-c-dcz9p is degraded
     nodes: build0-gstfj-ci-tests-worker-c-dcz9p
   Description: failed to drain node: build0-gstfj-ci-tests-worker-c-dcz9p after 1 hour. Please see machine-config-controller logs for more information
 
+Message: Cluster Version version is failing to reconcile (ClusterOperatorDegraded)
```

Comment:
The message reads a bit cryptic to me, let's figure out how to make it more informative to users.

Thinking loud: CV has a `Failing=True` condition (during upgrade) when it is failing to proceed with the update process for some reason. Can we  just say that? I do not think we want to mention "Cluster Version version" at all the CV resource is just a whiteboard CVO uses to write things on.

Reply:
I was pattern-matching [fmt.Sprintf("Cluster Operator %s is degraded (%s)", name, degraded.Reason)][1].  I could see dropping the `name` `%s` based on reasoning like "odds that the cluster has multiple ClusterVersions is very low, and it's not worth those low-signal characters in the vast majority of clusters where there is one just to decrease ambiguity in the zero-to-few clusters where there are multiple ClusterVersions".

I'm agnostic on "failing to proceed with the update" instead of "failing to reconcile".  To me those mean the same thing, but possibly one or the other is more likely to be clear to more admins?

[1]: https://github.com/openshift/oc/blob/958116d19854df93308103afaae8a3da8af7d31a/pkg/cli/admin/upgrade/status/controlplane.go#L86

Reply:
Remembering my time at TP I think I would understand "failing to proceed with the update" much better than "[CVO] is failing to reconcile [a ClusterVersion]", because it does not require me to understand that "cluster upgrade" is actually CVO reconciling individual manifests (we did not have great docs about that until recently) and using CV resource (with a specific name, knowing it is a singleton) as spec/status API for that.

So I'd vote for changing this message to something that clearly says "cluster update is stuck because of this".

Reply:
Remembering my time at TP I think I would understand "failing to proceed with the update" much better than "[CVO] is failing to reconcile [a ClusterVersion]", because it does not require me to understand that "cluster upgrade" is actually CVO reconciling individual manifests (we did not have great docs about that until recently) and using CV resource (with a specific name, knowing it is a singleton) as spec/status API for that (I think we also discouraged folks from touching `ClusterVersion` directly and rely on `oc` interpreting it, so we have some prior art in thinking that CV is closer to an internal detail than something admin should know about.

So I'd vote for changing this message to something that clearly says "cluster update is stuck because of this". I think the "failing to proceed with the update" is actually pretty good, maybe "CVO is failing to proceed with the update". I would keep the CV in the `Resources` field though, would just change the message only.

Reply:
Pivoted to failing to `proceed with the update` with 50e76ea90 -> 7977ec94e.